mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cfg/HibernateOrmMapperOutboxPollingSettings.java

```diff
@@ -514,16 +530,17 @@ private CoordinationRadicals() {
 
 		public static final String ENTITY_MAPPING_PREFIX = "entity.mapping.";
 		public static final String ENTITY_MAPPING_AGENT_PREFIX = ENTITY_MAPPING_PREFIX + "agent.";
-		public static final String ENTITY_MAPPING_AGENT_GENERATOR = ENTITY_MAPPING_AGENT_PREFIX + "generator";
 		public static final String ENTITY_MAPPING_AGENT_TABLE = ENTITY_MAPPING_AGENT_PREFIX + "table";
 		public static final String ENTITY_MAPPING_AGENT_SCHEMA = ENTITY_MAPPING_AGENT_PREFIX + "schema";
 		public static final String ENTITY_MAPPING_AGENT_CATALOG = ENTITY_MAPPING_AGENT_PREFIX + "catalog";
+		public static final String ENTITY_MAPPING_AGENT_UUID_GEN_STRATEGY = ENTITY_MAPPING_AGENT_PREFIX + "uuid_gen_strategy";
+		public static final String ENTITY_MAPPING_AGENT_UUID_DATA_TYPE = ENTITY_MAPPING_AGENT_PREFIX + "uuid_type";
```

Comment:
Shouldn't these be renamed `*_UUID_TYPE` instead of `*_UUID_DATA_TYPE`, since you renamed the key to simply `uuid_type`?

Obviously I'm not talking only about this occurrence, there are many others in this file and probably other files as well.